src/terms/anyonic.jl

```diff
@@ -54,7 +54,8 @@ function make_div_free(basis::PlaneWaveBasis{T}, A) where {T}
             out[1][iG], out[2][iG] = vec
         end
     end
-    [ifft(basis, out[α]) for α = 1:2]
+    # TODO: see https://github.com/JuliaMolSim/DFTK.jl/pull/722
```

Comment:
The PR does not really explain why this has been done. Either add explanation to the PR or put it into the code once and refer to it (better).